cli/src/pcluster/config/cluster_config.py

```diff
@@ -1574,8 +1574,6 @@ def _register_validators(self, context: ValidatorContext = None):  # noqa: D102
             ArchitectureOsValidator,
             os=self.image.os,
             architecture=self.head_node.architecture,
-            custom_ami=self.image.custom_ami,
-            ami_search_filters=self.dev_settings.ami_search_filters if self.dev_settings else None,
```

Comment:
Why is this removed?

Reply:
Because the validator was simplified. The validator was simplified because some checks on CentOS 7 are no longer needed.